modules/roles_profiles/manifests/profiles/gecko_1_b_osx_1015_generic_worker.pp

```diff
@@ -21,9 +21,9 @@
                 telegraf_user       => lookup('telegraf.user'),
                 telegraf_password   => lookup('telegraf.password'),
                 puppet_env          => 'dev',
-                puppet_repo         => 'https://github.com/davehouse/ronin_puppet.git',
+                puppet_repo         => 'https://github.com/mozilla-platform-ops/ronin_puppet.git',
                 puppet_branch       => 'bug1665379_mac-builders',
-                puppet_notify_email => 'dhouse@mozilla.com',
+                puppet_notify_email => 'relops@mozilla.com',
```

Comment:
If there is significant volume, let's move this to a different google group.

Reply:
I don't know honestly, I'll make a group.